TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -9,26 +9,20 @@ namespace TrafficManager.Manager.Impl {
     using JetBrains.Annotations;
     using TrafficManager.Util;
     using System.Reflection;
+    using TrafficManager.UI;
+    using TrafficManager.UI.Textures;
 
     public class OptionsManager
         : AbstractCustomManager,
           IOptionsManager {
+
         // TODO I contain ugly code
         public static OptionsManager Instance = new OptionsManager();
 
-        protected override void InternalPrintDebugInfo() {
-            base.InternalPrintDebugInfo();
-            Log.NotImpl("InternalPrintDebugInfo for OptionsManager");
-        }
-
-        /// <summary>
-        /// Converts value to SimulationAccuracy
-        /// </summary>
-        /// <param name="value">Old value</param>
-        /// <returns>SimulationAccuracy value</returns>
-        private static SimulationAccuracy ConvertToSimulationAccuracy(byte value) {
-            return SimulationAccuracy.MaxValue - value;
-        }
+        // See: OnAfterLoadData() and related methods
+        private static bool _needUpdateRoutingManager = false;
```

Comment:
I have a hunch it might not even be necessary (ie. it can probably be completely removed) but need to check first... brb

Reply:
`RoutingManager.OnAfterLoadData()` invokes `RecalculateAll()` so I don't need the `_needUpdateRoutingManager` stuff (during deserialisation, I can just ignore requests because post-deserialisation the `RoutingManager` is going to do full recalc anyway).

However I noticed that `RequestFullRecalculation()` is doing different stuff to `RecalculateAll()` (nothing to do with this PR) so that might merit some investigation at future date.